lib/eslint-tester.js

```diff
@@ -181,7 +188,16 @@ ESLintTester.prototype = {
         });
 
         return result.suite;
+    },
+
+    /**
+     * Gets the name of the last used parser.
+     * @returns {string} The name of the last used parser.
+     */
+    getParser: function() {
```

Comment:
OK, got it. All we really need to test is that the `parser` property is being set on the config that is passed to `eslint.verify()`. So you can use a Sinon spy to ensure that: http://sinonjs.org/docs/#spies